src/helm/proxy/models.py

```diff
@@ -866,6 +866,15 @@ def engine(self) -> str:
         "([paper](https://arxiv.org/pdf/2204.02311.pdf)).",
         tags=[TEXT_MODEL_TAG, LIMITED_FUNCTIONALITY_TEXT_MODEL_TAG],
     ),
+    # Nvidia
+    Model(
+        group="megatron",
+        creator_organization="Nvidia",
+        name="megatron/gpt2",
+        display_name="GPT-2",
+        description="GPT-2",
```

Comment:
"GPT-2" -> "Megatron GPT-2". Would be nice to have a longer description for description (can take from the GitHub readme or the paper abstract.